portfolio/src/main/webapp/script.js

```diff
@@ -65,31 +65,44 @@ function grow() {
 }
 
 function getComments() {
-  const commentsContainer = document.getElementById('comments-container');
-  // Clear all old comments
-  while (commentsContainer.firstChild) {
-    commentsContainer.firstChild.remove();
-  }
-  const limit = document.getElementById("limit").value;
+  const limit = document.getElementById('limit').value;
 
   fetch(`/data?limit=${limit}`)
     .then(response => response.json())
     .then(comments => {
+      const commentsContainer = document.getElementById('comments-container');
+      while (commentsContainer.firstChild) {
+        commentsContainer.firstChild.remove();
+      }
```

Comment:
Nit: Idk why there doesn't exist a method that removes all child elements. Since you have the same code below, why not make it into a helper method that accepts an element and removes all children?

Reply:
We can clear all child elements with `commentContainer.innerHTML=''`. I suppose it is not bad practice if it doesn't involve user input here?